### PR TITLE
ROX-18077: Delete Network Graph 1.0 in Components and Containers

### PR DESCRIPTION
**Pull request by pedrottimark**
## Description

In addition to imports from Components into Containers, there are also a few imports from Containers into Components.

Therefore, delete both folders at the same time, and then follow up to delete other files elsewhere whose only imports were from those folders.

From stackrox folder:

```sh
git rm -r ui/apps/platform/src/Components/DetailsOverlay
git rm -r ui/apps/platform/src/Components/NetworkEntityTabbedOverlay
git rm -r ui/apps/platform/src/Components/NetworkGraph
git rm -r ui/apps/platform/src/Containers/Network
```

### imports of networkTypes

Solutions to errors because of imports from Network Graph 1.0 into other folders:

1. `import { NetworkPolicyModification } from 'Containers/Network/networkTypes';`
    * src/Containers/NetworkGraph/api/useFetchBaselineNetworkPolicy.ts
    * src/Containers/NetworkGraph/hooks/useNetworkPolicySimulator.ts
    * src/Containers/NetworkGraph/simulation/NotifyYAMLModal.tsx
    * src/Containers/NetworkGraph/utils/simulatorUtils.ts

2. `import { EntityType } from 'Containers/Network/networkTypes';`
    * ui/apps/platform/src/Containers/NetworkGraph/utils/flowUtils.ts

3. `import { PortsAndProtocols, Edge } from 'Containers/Network/networkTypes';`
    * src/utils/networkUtils/getNetworkFlows.ts
    * delete src/utils/networkUtils folder and also the only reference in networkGraphUtils.test.js

4. `import { Edge } from 'Containers/Network/networkTypes';`
    * ui/apps/platform/src/utils/networkGraphUtils.test.constants.ts delete `deploymentEdges` because only reference deleted in preceding step.

### imports into reducers

Very temporary solutions to errors because of imports from Network Graph 1.0 into reducers:

1. `import { BaselineComparisonsResponse } from 'Containers/Network/networkTypes';`
    * src/reducers/network/baselineSimulation.ts
2. `import dialogueStages from 'Containers/Network/Dialogue/dialogueStages';`
    * src/reducers/network/dialogue.js
3. `import sidepanelStages from 'Containers/Network/SidePanel/sidepanelStages';`
    * src/reducers/network/sidepanel.js

### Learnings

1. Declare backend types outside of container:
    * src/services/WhatEverService.ts corresponds to proto/api/v1/what_ever_service.proto
    * src/types/whatEver.proto.ts corresponds to proto/storage/what_ever.proto
2. Make very clear by type names any processing changes to backend responses.

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

## Testing Performed

TL;DR only slight reduction in main.js because webpack understood deletion of async import in #6658

1. `yarn lint` in ui to find the above errors
2. `yarn build` in ui
    * `wc build/static/css/*.css`
        branch - master total: 0 = 5796730 - 5796730
        branch - master main.css: 0 = 4974847 - 4974847
    * `ls -al build/static/css/*.css | wc`
        branch - master: 0 = 27 - 27 files
    * `wc build/static/js/*.js`
        branch - master total: -255 = 11123451 - 11123706
        branch - master main.js: -255 = 4857197 - 4857452
    * `ls -al build/static/js/*.js | wc`
        branch - master: 0 = 75 - 75 files
    * `wc build/static/media/*.*`
        branch - master: 0 = 4476497 - 4476497
    * `ls -al build/static/media | wc`
        branch - master: 0 = 144 - 144 files
3. `yarn start` in ui

### Manual testing

1. Visit /main/dashboard

2. Click **Network Graph** in left navigation to visit /main/network-graph

3. Continue to visit every link in left navigation

### Integration testing

1. `yarn cypress-open` in ui/apps/platform
    * networkGraphPF/networkDeploymentSidebar.test.js
    * networkGraphPF/networkGraphPF.test.js

### CODE REVIEW

**Comment by pedrottimark** [comment]
ocp-4-10-ui-e2e-tests failure because pod restarts: collector

**Review by dvail — APPROVED** [review]
Thanks Mark! I agree with your summary under "Learnings" as well.
